lib/ssl/test/ssl_cert_SUITE.erl

```diff
@@ -1287,6 +1290,41 @@ basic_rsa_1024(Config) ->
                   {fail_if_no_peer_cert, true} | ServerOpts1],
     ssl_test_lib:basic_test(ClientOpts, ServerOpts, Config).
 
+%%--------------------------------------------------------------------
+server_certificate_authorities_disabled() ->
+     [{doc,"TLS 1.3: Disabling certificate_authorities extension on the server when verify_peer is set to true" 
+       " allows client auth with empty chain"}].
```

Comment:
I think we need to change the comment to say something like  "allows the client to send a chain that could be verifiable by the server but that would not adhere to the certificate_authorities extension as it is not part of the regular trusted certificate set"